ENS Rainbow homepage (#275)
namehash/ensnode: docs/ensrainbow/README.md

> # Astro Starter Kit: Basics

Reviewer: ```suggestion
# ENSRainbow Landing Page
```

> ```sh

Reviewer: Let's remove this.

pnpm create astro@latest -- --template basics
```

> [![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/withastro/astro/tree/latest/examples/basics)

Reviewer: Let's please remove this generic stuff.

[![Open with CodeSandbox](https://assets.codesandbox.io/github/button-edit-lime.svg)](https://codesandbox.io/p/sandbox/github/withastro/astro/tree/latest/examples/basics)
[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/withastro/astro?devcontainer_path=.devcontainer/basics/devcontainer.json)

> > 🧑‍🚀 **Seasoned astronaut?** Delete this file. Have fun!

Reviewer: Let's remove this.

> ![just-the-basics](https://github.com/withastro/astro/assets/2244813/a0a5533c-a856-4198-8470-2d67b1d7c554)

Reviewer: Let's remove this.

> ## 🚀 Project Structure

Reviewer: Let's please remove this generic "Project Structure" section of the readme.

Author: Understood

Inside of your Astro project, you'll see the following folders and files:

```text
/
├── public/
│   └── favicon.svg
├── src/
│   ├── layouts/
│   │   └── Layout.astro
│   └── pages/
│       └── index.astro
└── package.json
```

To learn more about the folder structure of an Astro project, refer to [our guide on project structure](https://docs.astro.build/en/basics/project-structure/).

> ## 🧞 Commands

Reviewer: I like this "Commands" section. Let's keep it.

All commands are run from the root of the project, from a terminal:

| Command                   | Action                                           |
| :------------------------ | :----------------------------------------------- |
| `pnpm install`             | Installs dependencies                            |
| `pnpm dev`             | Starts local dev server at `localhost:4321`      |
| `pnpm build`           | Build your production site to `./dist/`          |
| `pnpm preview`         | Preview your build locally, before deploying     |
| `pnpm astro ...`       | Run CLI commands like `astro add`, `astro check` |
| `pnpm astro -- --help` | Get help using the Astro CLI                     |

> ## 👀 Want to learn more?

Reviewer: ```suggestion
##  Want to learn more about Astro?
```

> Feel free to check [our documentation](https://docs.astro.build) or jump into our [Discord server](https://astro.build/chat).

Reviewer: ```suggestion
Feel free to check the [Astro documentation](https://docs.astro.build).
```
